CHANGELOG.md

```diff
@@ -1,21 +1,14 @@
-# NEXT RELEASE
+# 2.3.2 Release notes
 
 ### Bugfixes
 * Fixed bug when encryption was used. It would cause crashes and corrupted data 
   (sometimes pieces of data from earlier commits could be seen). PR #2465.
+  (fixes https://github.com/realm/realm-core/issues/2383)
```

Comment:
We sometimes have multiple issues for the same PR (containing the _actual_ code changes). Therefore I suggest we only note the PR number in the `CHANGELOG.md` and refer to all fixed issues in the PR. This way everything is _reachable_ and the PR containing the changes becomes the "canonical" reference. The alternative is to list _all_ issues as well as the PR? This might actually be impossible (or become a mess) if we at some later point in time find out that additional issues were fixed by a PR (if we then want to keep the list updated).